src/platform-includes/performance/enable-tracing/react-native.mdx

```diff
@@ -1,6 +1,6 @@
 <Alert level="Info" title="Prior Versions">
 
-If you use a version of our SDK prior to version `3.0.0`, you will need to include the `ReactNativeTracing` integration to use automatic instrumentation. You do not need to do this in versions `3.0.0` and above.
+If you use a version of our SDK prior to version `3.0.0`, you will need to include the `ReactNativeTracing` integration to use automatic instrumentation. You do not need to do this in versions `3.0.0` and above. The `tracingOrigins` option was renamed to `tracePropagationTargets` in version `4.9.0`.
```

Comment:
Could go on the migration guide instead?

Reply:
Sure. It's actually already there. 

```suggestion
If you use a version of our SDK prior to version `3.0.0`, you will need to include the `ReactNativeTracing` integration to use automatic instrumentation. You do not need to do this in versions `3.0.0` and above.
```